### PR TITLE
Update iOS/mac app home screen

### PR DESCRIPTION
**Pull request by smalluban**
According to https://github.com/ghostery/todo-wiktor/issues/2 and https://xd.adobe.com/view/a288a63d-d763-4981-b1af-61a6f8950694-f7db/, the PR updates the home screen of the iOS and Mac apps.

* `Main.html` moved to resources for better DX - it is possible to omit Xcode and run the page locally (now, the paths are in sync with the file system)
* The iOS screen with the page has solid background color (top and bottom), so the gradient background is attached only to the mac version

UPDATED SCREENSHOTS:
Zrzut ekranu 2022-03-23 o 14 33 47
Zrzut ekranu 2022-03-23 o 14 33 45

### CODE REVIEW

**Review by philipp-classen — APPROVED** [review]

**Comment by smalluban** [comment]
I was able to make WebView fullscreen on iOS, so I turned on the gradient effect on both platforms.

**Comment by smalluban** [comment]
@chrmod wants to replace the "needs help?" section with a link to "know more" (a blog post). We need to wait for the details before we can merge it.

**Comment by smalluban** [comment]
@chrmod The link is updated, so I think we are good to go with the PR. Please add your final review.

Zrzut ekranu 2022-03-24 o 13 41 26

**Review by chrmod — APPROVED** [review]

**Pull request merged**
